torchgeo/datasets/agb_live_woody_density.py

```diff
@@ -80,14 +80,17 @@ def __init__(
             cache: if True, cache file handle to speed up repeated sampling
```

Comment:
@adamjstewart can you take a look at the docstring for `download? just above this line?

`download: if True, download dataset and store it in the root directory`

It may be hard to know what that root is

Reply:
I think the majority of users will still use a single string in `paths`, and the docs do say that `paths` is one or more root directories, so hopefully they can figure it out. If users report confusion we can make this more clear or add a better error message in a future patch release.